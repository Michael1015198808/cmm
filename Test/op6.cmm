int main()
{
    /*
    comment
    /*
    nested comment
    */
    int i = 1;
    int i = 1;
    error sentence;
    */
    int i = 1;
}
