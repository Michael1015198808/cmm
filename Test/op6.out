Error type B at Line 10: syntax error
